cpp/ql/src/experimental/Security/CWE/CWE-561/FindIncorrectlyUsedSwitch.ql

```diff
@@ -0,0 +1,137 @@
+/**
+ * @name Operator Find Incorrectly Used Switch
```

Comment:
I think this name is more correct. What do you think?
```suggestion
 * @name Incorrect switch statement
```

Reply:
Did you not like this suggestion, @ihsinme?

Reply:
thanks